traffic_ops_ort/testing/README.md

```diff
@@ -0,0 +1,102 @@
+Traffic Ops ORT RPM files for the build you wish to test using 
+this framework.  This test environment provides the necessary
+Docker containers used to support and execute the tests when
+given the necessary RPM's.  If you choose to not use the provided
+Docker containers you will need to provide the following resources:
+
+  - A running Traffic Ops with the installed release to be tested
+  - A running Postgres SQL database which is loaded with the proper
+    test data for the Traffic Ops release.
+  - An Apache Traffic Server host that has the installed release of
+    ORT to be tested against the release of Traffic Ops.
+  - A yum server configured to provide the test rpm's herein.
+  - A Traffic vault server.
+
+# Directory layout
+
+  - trafficcontrol/traffic_ops_ort/testing/docker:  has all the 
+    necessary files for running the test Docker containers.
+  - trafficcontrol/traffic_ops_ort/testing/ort-tests:  this directory.
+    contains all the go files used to run the ORT tests.
+
+# Setup.
+
+  1.  Build the Traffic Ops and Traffic Ops ORT RPM's that you wish
+      to test.  See the top level 'build' directory for building 
+      instructions. 
+  2.  Copy the Traffic Ops RPM to docker/trafffic_ops/traffic_ops.rpm 
+      (NOTE:  Use the file name 'traffic_ops.rpm')
+  3.  Copy the Traffic Ops ORT rpm to docker/ats_edge/traffic_ops_ort.rpm
+      (NOTE:  Use the file name 'traffic_ops_ort.rpm'
+  4.  Copy the recent copy of riak-2.2.3-1.el7.centos.x86_64.rpm to
+      docker/traffic_vault/riak-2.2.3-1.el7.centos.x86_64.rpm
+  5.  Copy an Apache Trafficserver rpm to 
+      docker/yumserver/test-rpms/trafficserver-$VERSION.$COMMIT-HASH.el7.x86_64.rpm
+      You will need to edit and adjust the trafficserver package value in
+      ort-tests/tc-fixtures.json to match the $VERSION.$COMMIT-HASH used in the name
+      of your RPM.  Search for '8.0.8-19.77cb23a' in the ort-tests/tc-fixtures.json 
+      and change the value to match the RPM version you choose to use.
+      For example, the current value in tc-fixtures.json is '8.0.8-19.77cb23a' and 
+      therefore the rpm file expected in 'docker/yumserver/test-rpms' is
+      'trafficserver-8.0.8-19.77cb23a.el7.x86_64.rpm'.
+  6.  The container Docker files have the usernames and passwords used in the various
+      containers ie, postgresql db, traffic_ops, and traffic_ops_ort.  The usernames
+      and passwords passed to the 't3c' executable in in the 
+      ort-tests/conf/docker-edge-cache.conf file.  Make sure that the usernames/passwords
+      in the Docker files match those in the t3c configuration file.
+      An example ort-tests/conf/edge-cache.conf file is provided should you choose to
+      use your own Traffic Ops and Postgresql environment.
+  7.  Build the Docker images and start the containers:
+      ``` 
+      # cd trafficcontrol/traffic_ops_ort/testing/docker
+      # docker-compose build
+      # docker-compose up -d
+      ```
```

Comment:
the `#` prompt sorta makes this look like they're comments. You can also get syntax highlighting with a specifier like `shell`:

```shell
cd trafficcontrol/traffic_ops_ort/testing/docker
docker-compose build
docker-compose up -d
```

Reply:
@ocket8888 Fixed, removed the '#'